package.json

```diff
@@ -101,7 +107,8 @@
   "babel": {
     "presets": [
       "es2015",
-      "react"
+      "react",
+      "stage-1"
```

Comment:
Its a leftover to be removed. I wanted to use Class properties to make the class definitions for the react component leaner, but enzyme didn't like it so i had to roll it back for the moment :(